File: CPP/Cap04/c04ex05.cpp
// c04ex05.cpp

#include <iostream>
using namespace std;

int main(void) {
  int Mes;

  cout << "Entre com um numero equivalente a um mes: ";
  cin >> Mes;
  cin.ignore();

  switch (Mes) {
    case 1:
      cout << "Janeiro\n";
      break;
    case 2:
      cout << "Fevereiro\n";
      break;
    case 3:
      cout << "Marco\n";
      break;
    case 4:
      cout << "Abril\n";
      break;
    case 5:
      cout << "Maio\n";
      break;
    case 6:
      cout << "Junho\n";
      break;
    case 7:
      cout << "Julho\n";
      break;
    case 8:
      cout << "Agosto\n";
      break;
    case 9:
      cout << "Setembro\n";
      break;
    case 10:
      cout << "Outubro\n";
      break;
    case 11:
      cout << "Novembro\n";
      break;
    case 12:
      cout << "Dezembro\n";
      break;
    default:
      cout << "Mes invalido\n";
      break;
  }

  cout << "\nTecle <Enter> para encerrar... ";
  cin.get();

  return 0;
}

File: DART/Cap04/c04ex05.dart
// c04ex05.dart

import 'dart:io';

void main() {
  int Mes = 0;

  stdout.write("Entre com um numero equivalente a um mes: ");
  Mes = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  switch (Mes) {
    case 1:
      print("Janeiro");
      break;
    case 2:
      print("Fevereiro");
      break;
    case 3:
      print("Marco");
      break;
    case 4:
      print("Abril");
      break;
    case 5:
      print("Maio");
      break;
    case 6:
      print("Junho");
      break;
    case 7:
      print("Julho");
      break;
    case 8:
      print("Agosto");
      break;
    case 9:
      print("Setembro");
      break;
    case 10:
      print("Outubro");
      break;
    case 11:
      print("Novembro");
      break;
    case 12:
      print("Dezembro");
      break;
    default:
      print("Mes invalido");
      break;
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
